src/app/ApplicationTemplate.Presentation/ViewModels/Diagnostics/Configuration/ConfigurationDebuggerViewModel.cs

```diff
@@ -66,6 +70,28 @@ private void OnConfigurationChanged(object configuration)
 		);
 	});
 
+	/// <summary>
+	/// Gets all possible values for the <see cref="SelectedKey"/> property.
+	/// </summary>
+	public string[] AllKeys
```

Comment:
Does this need to be a dynamic property?

Reply:
Yes because it's updated from the `Save` command.